public/admin/config.yml

```diff
@@ -770,3 +776,63 @@ collections:
                 name: content
                 widget: markdown
                 hint: subscore definition
+
+  - name: regionOverrides
+    label: Region Overrides
+    description: Block metrics, add metric disclaimers, etc.
+    format: json
+    files:
+      - name: regionOverrides
+        label: Region Overrides
+        file: src/cms-content/region-overrides/region-overrides.json
+        fields:
+          - name: overrides
+            label: Overrides
+            widget: list
+            fields:
+              - name: region
+                label: Region
+                hint: State code (2 letters) or FIPS code (2 or 5 digits)
+                widget: string
+                pattern: ['^([A-Z][A-Z]|\d{2}|\d{5})$', 'Must be a 2-letter state code or 2 or 5-digit FIPS code']
+              - name: include
+                label: Include
+                widget: select
+                default: region
+                options:
+                  - label: Region
+                    value: region
+                  - label: Subregions (counties / metros)
+                    value: subregions
+                  - label: Region and subregions (counties / metros)
+                    value: region-and-subregions
```

Comment:
Isn't this last option more like: `state and all its counties and metros` / `metro and all its counties`?